### PR TITLE
 [KUNTECH-2703] Remove all atomic classes

### PR DESCRIPTION
**Pull request by ghost**
ticket https://jira.xing.hh/browse/KUNTECH-2703

### CODE REVIEW

**Comment by ghost** [comment]
From all classes  being removed here, I checked it first by looking into kununum app-mykununu, app-profiles, app-reviews and kununu.com repos to make sure they were not in use, neither in components or Sass files.

Some of them are still being extended on this repo, but in v1. Is that a problem?

File: packages/kununu-theme-v2/scss/base/type.scss [hunk 1]
```diff
@@ -89,22 +89,12 @@ p {
...
-  font-size: $font-size-base;
-}
-
 // Still used in mykununu
 .text-sm {
   font-size: $font-size-sm;
 }
 
-// Still used in insights
+// Still used in mykununu and profiles specs
```
Comment (catarina-lopes-pt) [inline packages/kununu-theme-v2/scss/base/type.scss]: Still used in footer, that's why is only in specs

File: packages/kununu-theme-v2/scss/base/type.scss [hunk 2]
```diff
@@ -114,7 +104,7 @@ p {
 // Still used in profiles
 .text-right   { text-align: right; }
 
-// Still used in homepage, mykununu sidebar
+// Still used in mykununu sidebar
 .text-center  { text-align: center; }
```
Comment (catarina-lopes-pt) [inline packages/kununu-theme-v2/scss/base/type.scss]: Where? I can't find it...

Comment (ghost) [reply packages/kununu-theme-v2/scss/base/type.scss]: Only in this file: https://github.com/kununu/app-mykununu/blob/master/src/components/App/__snapshots__/App.spec.jsx.snap

It's just a snapshot, but I'm not sure it's safe to be removed tho.

File: packages/kununu-theme-v2/scss/base/type.scss (hunk 1 above)
Comment (ghost) [reply packages/kununu-theme-v2/scss/base/type.scss]: @clopes-ventureoak Keep it here them?

Comment (catarina-lopes-pt) [reply packages/kununu-theme-v2/scss/base/type.scss]: Change the comment...

File: packages/kununu-theme-v2/scss/base/type.scss (hunk 2 above)
Comment (catarina-lopes-pt) [reply packages/kununu-theme-v2/scss/base/type.scss]: Still can't find it... only `text-center-xs text-center-sm`

**Review by catarina-lopes-pt — APPROVED** [review]
LGTM

**Review by kaytcat — APPROVED** [review]
Don't forget to bump the package version, prob a minor change.

**Review by kaytcat — APPROVED** [review]
lgtm

**Pull request merged**